test/users.test.js

```diff
@@ -82,3 +84,43 @@ describe('User registration test, with their respective fields', () => {
       });
   });
 });
+
+describe('User sign in test, with their respective fields', () => {
+  it('should sign in with all the fields correctly', done => {
+    testCreate('yesica@wolox.co', 'shdfgs345').then(() => {
+      request(app)
+        .post('/users/sessions')
+        .send({ email: 'yesica@wolox.co', password: 'shdfgs345' })
+        .set('Accept', 'application/json')
+        .then(response => {
+          const token = {
+            token:
+              'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6Inllc2ljYUB3b2' +
+              'xveC5jbyJ9.W94vf6ymuks9qEsz-dDciig304QtAa7FeUjlNqwXaI8'
+          };
+          expect(response.statusCode).toBe(200);
+          console.log(response.text);
```

Comment:
what is doing this here? jaajaajaj 😂😂😂😂😂😂

Reply:
Sorry! 😂